framework/test/unit/hooks/use-screen-size-test.ts

```diff
@@ -0,0 +1,36 @@
+import { renderHook } from '@testing-library/react'
+import mediaQuery from 'css-mediaquery'
+import { expect } from 'chai'
+import { useScreenSize } from '../../../lib'
+
+const createMatchMedia = (width: number) => (query: string) => ({
+  matches: mediaQuery.match(query, { width }),
+  addListener: () => {},
+  removeListener: () => {},
+  onchange: () => {},
+  media: '',
+  addEventListener: () => {},
+  removeEventListener: () => {},
+  dispatchEvent: (_event: Event) => true,
+})
+
+describe('useScreenSize', () => {
+  it('should return "large" for large screens', () => {
+    window.matchMedia = createMatchMedia(1280)
+    const { result } = renderHook(() => useScreenSize())
+
+    expect(result.current).to.equal('large')
+  })
```

Comment:
This test prob not needed, the dependency as well. If you want to keep the test then atleast move the dependency as a dev dependency